cmd/swagger/commands/generate/spec.go

```diff
@@ -23,16 +6,17 @@ import (
 	"os"
 	"strings"
 
+	"github.com/go-swagger/go-swagger/codescan"
+
 	"github.com/go-openapi/loads"
 	"github.com/go-openapi/spec"
-	"github.com/go-swagger/go-swagger/scan"
 	"github.com/jessevdk/go-flags"
 	"gopkg.in/yaml.v3"
 )
 
 // SpecFile command to generate a swagger spec from a go application
 type SpecFile struct {
-	BasePath    string         `long:"base-path" short:"b" description:"the base path to use" default:"."`
+	WorkDir     string         `long:"work-dir" short:"w" description:"the base path to use" default:"."`
```

Comment:
Is this backward compatible?

Reply:
No it's not. Options changed when migration rom "scan" to "codescan" 5 years ago. BasePath has not been in use since the post-go1.11 days (5 years ago?). Leaving it without.